WalletWasabi.Fluent/Views/Wallets/Home/Tiles/PrivacyRing/PrivacyRingView.axaml

```diff
@@ -52,7 +52,7 @@
   <c:ContentArea Title="{Binding Title}"
                  EnableNext="True" NextContent="Done"
                  EnableCancel="False"
-                 EnableBack="True"
+                 EnableBack="False"
```

Comment:
It should be bound to the `EnableBack` property, and the value should be set in the ViewModel.

This same is true for the `EnableCancel`. It should be bound to `EnableCancel` and the method `SetupCancel()` should be called in the ViewModel's ctor. This actually happens, only the first param is not correct compared to the value in the axaml. 
Either fix it here and change the title of the PR or do it in a separate PR.

Reply:
I noticed that in other places in the code base, I will do that in a follow up PR.